packages/svelte/src/compiler/phases/2-analyze/visitors/Identifier.js

```diff
@@ -87,6 +84,13 @@ export function Identifier(node, context) {
 		}
 	}
 
+	// no binding means global, and we can't inline e.g. `<span>{location}</span>`
+	// because it could change between component renders. if there _is_ a
+	// binding and it is outside module scope, the expression cannot
+	// be inlined (TODO allow inlining in more cases,
+	// e.g. primitive consts)
```

Comment:
these lines are surprisingly short. maybe they can be combined?
```suggestion
	// be inlined (TODO allow inlining in more cases - e.g. primitive consts)
```